liver_imaging_analysis/models/lesion_segmentation.py

```diff
@@ -41,6 +42,8 @@
 import nibabel as nib
 from monai.handlers.utils import from_engine
 import argparse
+import logging
+logger = logging.getLogger(__name__)
```

Comment:
why aren't you returning the logger from setup_logger

Reply:
i have fixed this